### DISCUSSION TITLE
Remote notebook deployments leveraging github settings

### DISCUSSION THREAD

**Discussion by babjacd-pfj** [databrickslabs/dbx#264] [Q&A]
As shown on the page here regarding the public preview: https://docs.databricks.com/repos/jobs-remote-notebook.html

There now is the ability within the UI to leverage running a notebook from a branch in a repo. What I am curious about is the ability to achieve this same functionality within the deployment.yml.j2? Either I am doing it wrong or it may not be full supported yet, but when I mimic the "settings" structure in the yml an error is thrown regarding work flow and the expected value of "name" is missing. It seems upon parsing that dbx is expecting the name of the job at that level in the hierarchy which is what leads me to believe it may not be supported yet.

Is this a feature which may be coming in a future release? Or if it is available now is there any examples on how to do this?

**Comment by renardeinside** [comment]
hi @babjacd-pfj , please find the documentation example here - https://dbx.readthedocs.io/en/latest/examples/notebook_remote.html
